moninag/moninag/urls.py

```diff
@@ -19,6 +19,7 @@
 
 urlpatterns = [
     url(r'^', include('home.urls')),
+    url(r'^api_01/check/', include('check.urls')),
```

Comment:
Replace on `url(r'^api/1/check/', include('check.urls')),`
I have changed too for Service View.
Liubomyr suggested :slightly_smiling_face: